src/lib/Context.ts

```
@@ -97,11 +113,23 @@
     return createSignerFromFile(path)
   }
 
-  // TODO move this warning to a better place
-  console.log('[warning]: No keypair specified, using temporary noop-signer')
-  // create no-op signer if no key is specified
+  throw new Error('Keypair file not found at: ' + path + ' please check your config file or use the --keypair flag')
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)